lib/config/migrations/base/remove-property-migration.ts

```diff
@@ -13,7 +13,7 @@ export class RemovePropertyMigration extends AbstractMigration {
     this.propertyName = propertyName;
   }
 
-  override run(): void {
+  run(): void {
```

Comment:
Why override is no longer required?

Reply:
I decided that is not correct to use `override` keyword for methods, which is marked as `abstract` in parent class

Reply:
Can you check if we set the `noImplicitOverride` typescript option? It should be enabled and force the override usage.

Reply:
Yeah, it is already enabled 5 month ago

Reply:
I think we should leave the override, so we can't get into trouble in future

Reply:
Returned